src/http_fork_source.rs

```diff
@@ -91,6 +91,15 @@ impl ForkSource for HttpForkSource {
             .wrap_err("fork http client failed")
     }
 
+    fn get_transaction_details(
+        &self,
+        hash: H256,
+    ) -> eyre::Result<Option<zksync_types::api::TransactionDetails>> {
+        let client = self.create_client();
+        block_on(async move { client.get_transaction_details(hash).await })
```

Comment:
>We are not caching these responses due to their mutable status. We could cache when status == Finalized, but didn't seem worth it. wdyt?

Agreed, not worth it. We should add this comment here so future devs have the context as to why we chose not to cache the response

Reply:
:+1: added comment